install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -128,10 +128,8 @@ spec:
           - --zipkinAddress
         {{- if $.Values.global.tracer.zipkin.address }}
           - {{ $.Values.global.tracer.zipkin.address }}
-        {{- else if $.Values.global.istioNamespace }}
-          - zipkin.{{ $.Values.global.istioNamespace }}:9411
         {{- else }}
-          - zipkin:9411
+          - zipkin.{{ $.Release.Namespace }}:9411
```

Comment:
let's say if gateway is installed in tube-system and istio control plane is installed in Istio-system, would $.Release.Namespace to be able to determine Istio-system when gateway is being installed?  Don't understand how this can be determined, would like an explanation.

Reply:
`{{ $.Release.Namespace }}` refers to the control plane namespace, always.

If the user wants to install the gateway in a different location, he must override `values.namespace` in the gateway chart.